securebanking-openbanking-uk-rcs-api/src/test/java/com/forgerock/securebanking/openbanking/uk/rcs/converters/InternationalPaymentConsentDetailsConverterTest.java

```diff
@@ -37,7 +36,7 @@ public class InternationalPaymentConsentDetailsConverterTest {
     @Test
     public void shouldConvertConsentDetailsToInternationalPaymentConsentDetails() {
         // Given
-        JsonObject consentDetails = aValidInternationalPaymentConsentDetails(INTERNATIONAL_PAYMENT_INTENT_ID);
+        JsonObject consentDetails = aValidInternationalPaymentConsentDetails(INTERNATIONAL_SCHEDULED_PAYMENT_INTENT_ID);
```

Comment:
I'm confuse with that change, this has been done in `InternationalPaymentConsentDetailsConverterTest` but the expected implementation test was `InternationalScheduledPayment......`, It seems that we are overriding a existing test class.